presto-docs/src/main/sphinx/functions/array.rst

```diff
@@ -77,6 +77,15 @@ Array Functions
 
     Subsets array ``x`` starting from index ``start`` (or starting from the end if ``start`` is negative) with a length of ``length``.
 
-.. function:: sequence(start, end) -> array
+.. function:: sequence(start bigint, stop bigint) -> array<bigint>
 
-    Sequence function to generate synthetic arrays, starting from integer value start, to integer value end.
+    Generate a sequence of integers from `start` to `stop`, incrementing by 1 if `start <= stop` and -1 if `start > stop`.
+
+.. function:: sequence(start bigint, stop bigint, step bigint) -> array<bigint>
```

Comment:
Let's leave this the old way until we update all the docs. Better to be consistent.